packages/client/test/testdata/gethk5vecs/statediffs.json

```diff
@@ -1 +1,457 @@
0000000000000000000000000000000000000000000"},{"suffix":3,"currentValue":null,"newValue":"0xd1d29ee74a6d03244189ddb39239adc2a5f77ba91a8df459f17a172dbd96213d"},{"suffix":4,"currentValue":null,"newValue":"0x0a00000000000000000000000000000000000000000000000000000000000000"},{"suffix":128,"currentValue":null,"newValue":"0x0060606040526008565b00000000000000000000000000000000000000000000"}]},{"stem":"0xddb1869fe308ddab3660375687fd2a3f94434c961ed68fc8beb6fc8459771c","suffixDiffs":[{"suffix":0,"currentValue":null,"newValue":"0x0000000000000000000000000000000000000000000000000000000000000000"},{"suffix":1,"currentValue":null,"newValue":"0x0000000000000000000000000000000000000000000000000000000000000000"},{"suffix":2,"currentValue":null,"newValue":"0x0300000000000000000000000000000000000000000000000000000000000000"},{"suffix":3,"currentValue":null,"newValue":"0x855d6153e73150a38cab4b29ba925205be4f569cc2b5e049dfe5aca12d8ed260"},{"suffix":4,"currentValue":null,"newValue":"0xbd01000000000000000000000000000000000000000000000000000000000000"},{"suffix":64,"currentValue":null,"newValue":"0x0000000000000000000000000f2cc432c8892c12eede08774adfd63df6f49282"},{"suffix":65,"currentValue":null,"newValue":"0x00000000000000000000000074b186f98c406a19aaf31b542b2b2353fb1577f6"},{"suffix":128,"currentValue":null,"newValue":"0x00608060405234801561001057600080fd5b506004361061002b5760003560e0"},{"suffix":129,"currentValue":null,"newValue":"0x001c8063f566852414610030575b600080fd5b61003861004e565b6040516100"},{"suffix":130,"currentValue":null,"newValue":"0x01459190610146565b60405180910390f35b6000600160009054906101000a90"},{"suffix":131,"currentValue":null,"newValue":"0x000473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffff"},{"suffix":132,"currentValue":null,"newValue":"0x0dffffffffffffffffffffffffff166381ca91d36040518163ffffffff1660e0"},{"suffix":133,"currentValue":null,"newValue":"0x001b815260040160206040518083038186803b1580156100b857600080fd5b50"},{"suffix":134,"currentValue":null,"newValue":"0x005afa1580156100cc573d6000803e3d6000fd5b505050506040513d601f1960"},{"suffix":135,"currentValue":null,"newValue":"0x011f820116820180604052508101906100f0919061010a565b905090565b6000"},{"suffix":136,"currentValue":null,"newValue":"0x008151905061010481610170565b92915050565b600060208284031215610120"},{"suffix":137,"currentValue":null,"newValue":"0x005761011f61016b565b5b600061012e848285016100f5565b91505092915050"},{"suffix":138,"currentValue":null,"newValue":"0x00565b61014081610161565b82525050565b600060208201905061015b600083"},{"suffix":139,"currentValue":null,"newValue":"0x000184610137565b92915050565b6000819050919050565b600080fd5b610179"},{"suffix":140,"currentValue":null,"newValue":"0x0081610161565b811461018457600080fd5b5056fea2646970667358221220a6"},{"suffix":141,"currentValue":null,"newValue":"0x00a0e11af79f176f9c421b7b12f441356b25f6489b83d38cc828a701720b41f1"},{"suffix":142,"currentValue":null,"newValue":"0x1064736f6c634300080700330000000000000000000000000000000000000000"}]}]]
```

Comment:
Same thing here with linting updates from verkle testnet data.

Reply:
Would assume this is the same, just linting stuff as I didn't touch it directly.